templates/repo/view_file.tmpl

```diff
@@ -122,6 +130,8 @@
 					<a class="item copy-line-permalink" data-url="{{.Repository.Link}}/src/commit/{{PathEscape .CommitID}}/{{PathEscapeSegments .TreePath}}{{if $.HasSourceRenderedToggle}}?display=source{{end}}">{{.locale.Tr "repo.file_copy_permalink"}}</a>
 				</div>
 				{{end}}
+			{{else if not .FileSize}}
```

Comment:
If this check could be moved before all others, I guess a lot of `<div>empty</div>` could be eliminated

Reply:
That was my initial idea as well, but I wasn't sure whether alle code paths do set `.FileSize` correctly.